packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php

```diff
@@ -29,15 +31,23 @@ public function __construct(
      */
     public function create(): Configuration
     {
+        if (isset($this->configuration)) {
```

Comment:
why do not you use `InMemoryCache` service for the caching?

Reply:
I do not want to use ResetInterface for this as it is not necessary